File: Python/1-To-N-Even-Only.py
#  Question : To print even numbers from 1 to N
#        Input : User input a number
#         Output : Even natural numbers

n=int(input("enter the number : "))
i=2
while(i<=n):
    if(i%2==0):
        print(i,end='\n')
    i=i+1

File: C/1-TO-N-Even-Only.c
/* Question : To print even numbers from 1 to N
       Input : User input a number
       Output : Even natural numbers
*/

#include <stdio.h>
int main()
{
    int n, i=2;
    printf("Enter the value : ");
    scanf("%d",&n);
    while(i<=n)
    {
        if(i%2==0)
            printf("%d ",i);
        i=i+1;
    }
    return 0;
}